<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="textIP">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinPort">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>56789</number>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonConnect">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>611</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridGame"/>
   </widget>
   <widget class="QPushButton" name="buttonStart">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIP">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelVoters">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>551</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
